crates/gpui/src/platform/linux/window.rs

```diff
@@ -211,6 +212,12 @@ impl LinuxWindowState {
             .unwrap(),
         );
 
+        let gpu_extent_after_init = query_render_extent(&xcb_connection, x_window);
+        if gpu_extent != gpu_extent_after_init {
+            log::warn!("Gpu extent changed after gpu::Context::init_windowed");
```

Comment:
This is probably good for diagnostics, to see what distributions/hardware are affected by this